### ISSUE TITLE
EPACTS Zoom Plot

### ISSUE THREAD

**Issue by ana-stankovic** [statgen/EPACTS#21]
I am having problems in running Zoom Plot with the pre-release version 3.4.2.
The command line is following:
`epacts zoom --pos 20:87662 --prefix  _1_20_38_zoomTest --vcf unit_true_1000G_exome_chr20_example_softFiltered.calls.vcf_GRCh38_liftover.vcf.gz`
In the attachment is the error log file - [job.err.log](https://github.com/statgen/EPACTS/files/4606635/job.err.log)

The same command line works with the installed 3.3.0 version and with the same files, and a good plot is generated.
`/opt/EPACTS/bin/epacts-zoom zoom --pos 20:87662 --prefix  _1_20_38_zoomTest --vcf unit_true_1000G_exome_chr20_example_softFiltered.calls.vcf_GRCh38_liftover.vcf`

Do you have some insights in the changes in the new release that could cause this problem?
Thank you

**Comment by jonathonl** [comment]
I'll look into it. Thanks for reporting. Is there a corresponding csi or tbi index for your vcf file?

**Comment by ana-stankovic** [comment]
Yes, there is the .tbi index and the VCF is gzipped. The identical test files were used for both runs, with the only difference in the version of the EPACTS.
The variant is also present in the VCF file.
Thank you for the fast responses.
